Languages/ru/README.md

```diff
@@ -0,0 +1,143 @@
+
+### Поддерживаемые платформы ###
+OpenTomb - кросс-платформенный движок: в настоящее время он может запускаться в Windows, Mac или Linux. Никаких мобильных реализаций пока нет, но они по сути возможны.
+
+### Установка ###
+Чтобы запустить любой из уровней из исходных игр, вам понадобятся активы из соответствующей игры. Эти ресурсы часто имеют тенденцию находится в загадочных форматах, вариаций в играх. Из-за этого вам нужно будет конвертировать некоторые игровые ресурсы к используемым форматам самостоятельно или получить их откуда-то в Сети.
+
+Вот список всех необходимых активов и где их можно получить:
+
+ * Папки данных из каждой игры. Получите их с компакт-дисков вашей приобретенной игры или Steam / GOG узлы. Просто возьмите папку с данными из папки каждой игры и поместите ее в соответствующей папке `/ data / tr * /`. Например, для TR3 путь будет `OpenTomb / данные / TR3 / данные /`
+
+ * Аудиодорожки CD. OpenTomb поддерживает только OGG мгновенные аудиотреки, так что вы должны преобразовать оригинальные саундтреки самостоятельно или просто загрузить весь TR1-5 музыкальный пакет здесь: https://opentomb.earvillage.net ПОЖАЛУЙСТА, ОБРАТИТЕ ВНИМАНИЕ: файлы, возможно, потребуется переименовать, чтобы все работало, подробнее https://github.com/opentomb/OpenTomb/issues/447
+
+ * Загрузка экранов для TR1-3 и TR5. Для TR3, извлечь их из директории pix
+ вашей установленной официальной игры. Просто поместите этот каталог пиков в `/ data / tr3 /`папку. Что касается других игр, немного сложнее получить загрузочные экраны, так как там не было загрузочных экранов для версий ПК TR1-2, TR4 использовали скриншоты уровней как загрузочные экраны и TR5 использовали зашифрованный формат для хранения всей загрузки графики. Итак, чтобы облегчить вашу жизнь, вы можете просто загрузить загрузочный экран здесь: http://trep.trlevel.de/temp/loading_screens.zip
+ Просто поместите это прямо в каталог OpenTomb, и это должно быть хитростью. Заметка: движок поддерживает png и pcx формат изображений на экране.
+
+### Компиляция ###
+Файл CMakeLists.txt имеет исходный код, поэтому вы можете компилировать OpenTomb с использованием CMake. В Windows вы также можете скомпилировать его из Code ID Blocks IDE (файл проекта также предоставляется). Кроме того, вы можете вручную скомпилировать его в Код::Блоки рекурсивного добавления всех исходных файлов из каталога / src и добавив эти библиотеки в настройки компоновщика в разделе «Параметры сборки проекта»:
+
+* libmingw32.a
+* libSDL2main.a
+* libSDL2.dll.a
+* liblua.a
+* libpng.a
+* libz.a
+* libpthread.a
+
+В Linux просто загрузите исходный код и запустите в терминале:
+
+    cmake . && make
+
+Необходимыми зависимостями являются заголовки разработки для SDL2, png, LUA 5.2, ZLIB. Вы можете установить их в дистрибутив на основе Ubuntu с помощью этой команды:
+
+    sudo apt-get install libopenal-dev libsdl2-dev libpng12-dev liblua5.2-dev libglu1-mesa-dev zlib1g-dev
```

Comment:
liblua5.2-dev - outdated, forget to remove, now LUA 5.3 is used and it is internal (just delete liblua5.2-dev)